### PR TITLE
refactor: utils

### PR DESCRIPTION
**Pull request by evilebottnawi**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

Move utils to lib for better testing ans solve future problem when `open`/`bonjure` doesn't work without CLI

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

File: bin/webpack-dev-server.js [hunk 1]
```diff
@@ -4,14 +4,8 @@
...
-  import/no-extraneous-dependencies,
-  global-require,
   no-shadow,
-  no-console,
-  multiline-ternary,
-  arrow-parens,
-  array-bracket-spacing,
-  space-before-function-paren
+  no-console
 */
```
Comment (evilebottnawi) [inline bin/webpack-dev-server.js]: Remove `eslint` comments around stylistics rules, we use `prettier` :+1:

File: bin/webpack-dev-server.js [hunk 2]
```diff
@@ -26,14 +20,16 @@ const webpack = require('webpack');
...
 
 const addEntries = require('../lib/utils/addEntries');
+const colors = require('../lib/utils/colors');
+const createConfig = require('../lib/utils/createConfig');
 const createDomain = require('../lib/utils/createDomain');
 const createLogger = require('../lib/utils/createLogger');
-const createConfig = require('../lib/utils/createConfig');
+const getVersions = require('../lib/utils/getVersions');
+const runBonjour = require('../lib/utils/runBonjour');
+const status = require('../lib/utils/status');
```
Comment (evilebottnawi) [inline bin/webpack-dev-server.js]: Move this to utils, we should testing their, let's do it in other PR

File: lib/Server.js [hunk 3]
```diff
@@ -702,7 +699,10 @@ class Server {
       return true;
     }
 
-    if (!headerToCheck) headerToCheck = 'host';
+    if (!headerToCheck) {
+      headerToCheck = 'host';
+    }
```
Comment (evilebottnawi) [inline lib/Server.js]: Add `{` and `}` to all construction (better reading), we should enforce this rule in eslint, let's do it in other PR

**Comment by evilebottnawi** [comment]
No changes, only refactor

File: bin/webpack-dev-server.js (hunk 2 above)
Comment (hiroppy) [reply bin/webpack-dev-server.js]: Thank you for separating to utils 🙏

**Comment by hiroppy** [comment]
CI failed https://ci.appveyor.com/project/sokra/webpack-dev-server/builds/22574490/job/7wk6h9jwhrisjcvt

**Pull request merged**